app/status.go

```diff
@@ -137,7 +137,9 @@ func GetUserStatusesByIds(userIds []string) ([]*model.Status, *model.AppError) {
 		}
 	}
 	for _, userId := range missingUserIds {
-		statusMap = append(statusMap, &model.Status{UserId: userId, Status: "offline"})
+		if result := <-Srv.Store.User().Get(userId); result.Err == nil {
+			statusMap = append(statusMap, &model.Status{UserId: userId, Status: "offline"})
```

Comment:
i will talk with you the the chat, there are some behavior I would like to discuss